modules/ROOT/pages/notifications-configuration-reference.adoc

```diff
@@ -0,0 +1,113 @@
+|`notification-listener` |0..* |Registers a bean as a listener with the notification system. Events are dispatched by reflection, which means that the listener receives all events associated with the interfaces it implements.
+|===
+
+== Notification
+
+Associates an event with an interface. Listeners that implement the interface will receive instances of the event.
+
+=== Attributes of Notification
+
+[%header%autowidth.spread]
+|===
+|Name |Description
+|`event-class` |The class name of the notification event to deliver to the interface. You can use this instead of the `event` attribute to specify a custom class. +
+*Type*: `class name` +
+*Required*: no +
+*Default*: none
+|`event` |The notification event to deliver. +
+*Type*: `notificationTypes` +
+*Required*: no +
+*Default*: none
+|`interface-class` |The class name of the interface that receives the notification event. +
+*Type*: `class name` +
+*Required*: no +
+*Default*: none
+|`interface` |The name of the interface that receives the notification event. +
+*Type*: `notificationTypes` +
+*Required*: no +
+*Default*: none
+|===
+
+There are no child elements for `notification`.
+
+== Disable Notification
+
+Blocks the association of an event with a particular interface. This filters events after the association is made with the interface, so it takes precedence over over other elements, such as `notification-listener`.
+
+=== Attributes of disable-notification
+
+[%header%autowidth.spread]
+|===
+|Name |Description
+|`event-class` |The class name of the event that is no longer delivered to an interface. You can use this instead of the `event` attribute to specify a custom class. +
+*Type*: `class name` +
+*Required*: no +
+*Default*: none
+|`event` |The event you no longer want to deliver. +
+*Type*: `notificationTypes` +
+*Required*: no +
+*Default*: none
+|`interface-class` |The class name of the interface that no longer receives the event. +
+*Type*: `class name` +
+*Required*: no +
+*Default*: none
+|`interface` |The name of the interface that no longer receives the event. +
+*Type*: `notificationTypes` +
+*Required*: no +
+*Default*: none
+|===
+
+There are no child elements for `disable-notification`.
+
+== Notification Listener
+
+Registers a bean as a listener with the notification system. Events are dispatched by reflection, which means that the listener receives all events associated with the interfaces it implements.
+
+== Notification Types
+
+You can specify the following types of notifications using the `event` attribute of the `notification` and `disable-notification` elements:
+
+* ASYNC-MESSAGE
+* CONNECTION
+* CONNECTOR-MESSAGE
+* CUSTOM
+* EXCEPTION
+* EXCEPTION-STRATEGY
+* MANAGEMENT
+* MESSAGE-PROCESSOR
+* PIPELINE-MESSAGE
+* REGISTRY
+* ROUTING
+* SECURITY
+* TRANSACTION
```

Comment:
Each of these values in the list should be enclosed with backticks

Reply:
OK